addons/project/views/project_views.xml

```diff
@@ -1366,19 +1357,25 @@
                                     </div>
                                 </div>
                                 <div class="o_kanban_record_bottom" t-if="!selection_mode">
-                                    <div class="oe_kanban_bottom_left">
-                                        <field name="priority" widget="priority"/>
-                                        <field name="activity_ids" widget="kanban_activity"/>
+                                    <div class="oe_kanban_bottom_left" t-attf-class="{{record.is_closed.raw_value ? 'opacity-50' : ''}}">
+                                        <field name="priority" widget="priority" style="margin-right: 5px;"/>
+                                        <field name="activity_ids" widget="kanban_activity" style="padding-top: 1.5px;"/>
                                         <b t-if="record.rating_active.raw_value and record.rating_count.raw_value &gt; 0" groups="project.group_project_rating">
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
+                                            <span style="font-weight:bold; padding-top:1.5px; padding-right:3px; font-size:16px;" class="fa fa-fw fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
+                                            <span style="font-weight:bold; padding-top:1.5px; padding-right:3px; font-size:16px;" class="fa fa-fw fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
+                                            <span style="font-weight:bold; padding-top:1.5px; padding-right:3px; font-size:16px;" class="fa fa-fw fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
                                         </b>
-                                        <a t-if="record.subtask_count.raw_value and record.allow_subtasks.raw_value" class="subtask_list_button" title="See Subtasks" style="padding: 0px 5px 0px 0px; box-sizing: border-box; font-size: 20px;"/>
+                                        <a t-if="!record.project_id.raw_value" class="text-muted" style="font-size: 17px; padding-top: 1.5px;"><i title="Private Task" class="fa fa-lock"></i></a>
```

Comment:
```suggestion
                                        <a t-if="!record.project_id.raw_value" class="text-muted" style="font-size: 17px; padding-top: 1.5px;">
                                            <i title="Private Task" class="fa fa-lock"/>
                                        </a>
```
- New lines
- <i> tag can be directly closed, can't it?

Reply:
I'll work on this